libs/live-fns/keybase.js

```diff
@@ -0,0 +1,29 @@
+const got = require('../got.js')
+
+const getPGPFingerprint = user => {
```

Comment:
I think the function should only convert the fingerprint to 64-bit. The `user.public_keys.primary.key_fingerprint` part should just be in the main function. Alternatively, it should be split into two functions.

Reply:
Ok, I split it up like that because we need to check if `user` is null or not first. So I was planning on implementing that check, and then just passing the user object over to the function if all is good.

However, it seems like Badgen is automatically falling back to the default if anything throws so I didn't end up implementing a check.

Reply:
You mean like this?

https://github.com/amio/badgen-service/pull/141/commits/c27be549e8f2d5d0d3112effc91137f80f3ceca6

Reply:
Yes, but rename the function to something like `convertFingerprintTo64bit`.

Reply:
Done 👌